dist/steps/build.sh

```diff
@@ -143,6 +143,19 @@ if [[ "$BUILD_TARGET" == "StandaloneOSX" ]]; then
   chmod +x $ADD_PERMISSIONS_PATH
 fi
 
+
+if [[ -n "$CHOWN_FILES_TO" ]]; then
+  if [[ -n "$BUILD_PATH_FULL" ]]; then
+    chown -R $CHOWN_FILES_TO $BUILD_PATH_FULL
+  fi
+  if [[ -n "$UNITY_PROJECT_PATH" ]]; then
+    chown -R $CHOWN_FILES_TO $UNITY_PROJECT_PATH
+  fi
+  if [[ -n "$CUSTOM_BUILD_PATH" ]]; then
+    chown -R $CHOWN_FILES_TO $CUSTOM_BUILD_PATH
+  fi
+fi
+
```

Comment:
this can be simplified to 

```suggestion
if [[ -n "$CHOWN_FILES_TO" ]]; then
  chown -R $CHOWN_FILES_TO $BUILD_PATH_FULL
fi

```

Reply:
The build path may be somewhere completely different to the project path, and the project path has Library and Logs directories created and owned by root, as well as the UnityBuilderAction files copied into the project 

Chowning $CUSTOM_BUILD_PATH might be redundant though indeed

Reply:
Right unity project path will also be needed. Editing suggestion